packages/deploy/src/validator.ts

```diff
@@ -50,14 +53,23 @@ export function parseAndValidate(input: string): {
     if (typeof workflows === 'undefined') {
       // allow workflows to be unspecified, but ensure there is an empty
       // map to avoid errors downstream
-      doc.setIn(['workflows'], {})
-    }
-    else if (isMap(workflows)) {
+      doc.setIn(['workflows'], {});
+    } else if (isMap(workflows)) {
```

Comment:
Please declare the `workflowKeys` array here, inside this if statement (but outside of the for, obviously).

That scopes the variable to where it's actually used, which feels neater